Model/Product/Repository.php

```diff
@@ -312,11 +305,7 @@ private function saveParentIdsToCache(ProductInterface $product, $parentProductI
      */
     public function getSkusAsArray(Product $product, Store $store)
     {
-        $inStockProductsByIds = $this->stockProvider->getInStockProductIds(
```

Comment:
This will introduce another bug where the product price would be incorrect if the cheapest sku is out-of-stock

Reply:
@olsi-qose ah, so the "as low as" price is dynamically changing in Nosto depending on the stock status?